bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZATo2LA01j_5f_pta130_NLO_FXFX/ZATo2LA01j_5f_pta130_NLO_FXFX_proc_card.dat

```diff
@@ -0,0 +1,12 @@
+#switch to diagonal ckm matrix if needed for speed 
+import model loop_sm-no_b_mass 
+ 
+define p = p b b~ 
```

Comment:
too fast... can you please remove the proton and jet definitions and open another pr?

Reply:
sorry I missed this comment, will do now!